### PR TITLE
LPS-187912 Fixing infoFieldType to be able to filter by ImageInfoFieldType when mapping fields in image fragment

### PR DESCRIPTION
**Pull request by YurenaC**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-187912)

https://github.com/liferay-echo/liferay-portal/assets/85171101/3fc3e878-00cd-424e-8410-c2b8b98ef5e6

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/ObjectEntryInfoItemFields.java [hunk 1]
```diff
@@ -80,6 +80,15 @@ public class ObjectEntryInfoItemFields {
 		).labelInfoLocalizedValue(
 			InfoLocalizedValue.localize(ObjectEntryInfoItemFields.class, "id")
 		).build();
+	public static final InfoField<ImageInfoFieldType> previewURLInfoField =
```
Comment (ealonso) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/ObjectEntryInfoItemFields.java:83]: @YurenaC Where are we using this static field?

Comment (YurenaC) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/ObjectEntryInfoItemFields.java]: Nowhere, my bad

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**